root/release/edit/information.tt

```diff
@@ -236,7 +236,7 @@
         </td>
         <td><label>[% l('Cat. No:') %]</label></td>
         <td>
-          <input type="text" data-bind="withLabel: 'catno', value: catalogNumber, valueUpdate: 'input', controlsBubble: $root.labelBubble" />
+          <input type="text" data-bind="withLabel: 'catno', value: catalogNumber, valueUpdate: 'input', controlsBubble: $root.catalogNumberBubble" />
```

Comment:
I created a separate bubble for the catalog number since it was previously sharing the label bubble.